app/controllers/api/providers_controller.rb

```diff
@@ -85,6 +85,21 @@ def import_vm_resource(type, id = nil, data = {})
       end
     end
 
+    def options
+      if params[:ems_type].blank?
```

Comment:
It is not a must, but I though it will be nice. I don't mind documenting it (where?) but I can also remove this.